src/app/components/image-carousel/image-carousel.component.ts

```diff
@@ -9,32 +10,48 @@ import {FileService} from '../../admin-view/services/file-service';
 })
 export class ImageCarouselComponent implements OnInit {
 
-  @Input() carouselItems: Array<string>;
+  @Input() carouselItems: Array<string> | Array<CarouselProject>;
+  public withText: boolean;
```

Comment:
daj tento public atribut nizsie tam kde su ostatne public atributy nech sa to nemiesa tu so vstupmi